pallets/avn/src/tests/tests.rs

```diff
@@ -1,64 +1,91 @@
-use crate::mock::*;
+use crate::{mock::*, OperationType};
```

Comment:
Overall, I think these tests (even though they're clearly unit tests) could be written in a more BDD style:

- when `calculate_primary_validator` is called for operation Ethereum
  - the next validator for Ethereum increments by one, if the current one is not the last
  - the validator for Ethereum wraps around, if the current one is the last
  - the validator for AvN stays the same

- calling `is_primary_validator` for Ethereum
  - does not change the next_primary validator for Ethereum
  - returns true if the argument is the same as the next Ethereum validator
  - returns false if it is not

(repeat for the other operation)